src/Avalonia.Base/Styling/TypeNameAndClassSelector.cs

```diff
@@ -15,7 +14,7 @@ namespace Avalonia.Styling
     internal class TypeNameAndClassSelector : Selector
     {
         private readonly Selector? _previous;
-        private readonly Lazy<List<string>> _classes = new Lazy<List<string>>(() => new List<string>());
+        private List<string>? _classes;
```

Comment:
@grokys was the Lazy important?
@maxkatz6

Reply:
It's replaced with `??=` below